x-pack/plugins/reporting/README.md

```diff
@@ -10,4 +10,11 @@ This should be deprecated. It is historically a customer driven endpoint.  This
 This new endpoint is designed to have a more automation-friendly signature. It will replace csv_searchsource in the UI at some point, when there is more capacity in reporting. It will need a little more work to have parity: it needs to be able to export "unsaved" searches.
 
 ## Generate CSV
-Although historically related to reporting, the CsvGenerator class has now be moved into its own package `@kbn/generate-csv` and `@kbn/generate-csv-types`. 
\ No newline at end of file
+Although historically related to reporting, the CsvGenerator class has now be moved into its own package `@kbn/generate-csv` and `@kbn/generate-csv-types`. 
+
+## Serverless configuration
+There are several improvements made for reporting in serverless environments. Most changes are reflected in `reporting/server/config/schema.ts` for reference. 
+
+PNG and PDF reports are not possible in serverless. CSV reporting is enabled by default but can be set to `xpack.reporting.export_types.csv.enabled: false`. Image reporting will not be registered by the export types registry in the reporting plugin if not enabled in the config.
```

Comment:
We can leave out the sentence about turning off CSV, since the config is not easily editable for serverless.

```suggestion
PNG and PDF reports are currently not possible in serverless. Those export types are not enabled in serverless configuration, and are left out of Reporting's internal registry of export types.
```